src/explanations/sushiswap.yaml

```diff
@@ -0,0 +1,130 @@
+
+
+    Additionally, Limit Order v2 introduces a new feature called "order batching," which allows multiple orders to be processed together in a single transaction. This reduces gas costs and increases the speed of order processing. Overall, Limit Order v2 aims to provide a more efficient and cost-effective way for users to place limit orders on the Sushi decentralized exchange.
+
+
+    #### Masterchef v2
+
+    Masterchef v2 is a feature that allows users to automate the creation of liquidity pools on SushiSwap. A liquidity pool is a pool of assets that users can contribute in exchange for a share of the trading fees. By automating the creation of liquidity pools, Masterchef v2 aims to make it easier for users to provide liquidity and earn rewards on the platform.
+
+
+    Masterchef v2 introduces several improvements over the original Masterchef feature, including the ability to create liquidity pools for any ERC-20 token and the ability to adjust the weight of each asset in the pool. It also introduces a new feature called "flex pools," which allow users to provide liquidity for multiple assets within a single pool. This allows users to earn rewards on multiple assets at once, rather than having to create separate pools for each asset. Overall, Masterchef v2 aims to provide a more flexible and efficient way for users to provide liquidity on the SushiSwap platform.
+
+
+    #### SushiSwap
+
+    SushiSwap is a decentralized exchange (DEX) that allows users to buy and sell cryptocurrencies and tokens. It is built on the Ethereum blockchain and utilizes the Automated Market Maker (AMM) model for pricing and liquidity. AMM exchanges do not have traditional order books, but rather use algorithmic pricing based on the supply and demand of assets in the exchange.
+
+
+    One of the key features of SushiSwap is its liquidity pools, which allow users to provide liquidity in exchange for a share of the trading fees. Users can contribute assets to a liquidity pool and receive a share of the pool proportional to their contribution. These pools provide liquidity for trades on the SushiSwap DEX and allow users to earn rewards for providing liquidity.
+
+
+    SushiSwap also includes a feature called "SushiBar," which allows users to earn yield on their idle tokens by serving them up for others to use in trades on the DEX. SushiSwap aims to provide a decentralized and efficient platform for buying and selling cryptocurrencies and tokens.
+
+    #### Trident AMM Framework
+
+    Trident is a framework for building and deploying automated market makers (AMMs) on the Ethereum blockchain. Unlike traditional AMMs, which are hard-coded pools with specific features, Trident is a framework that allows users to create any type of AMM they desire. It is based on the concept of "interface-based programming," which allows developers to rapidly and collectively build new AMM pool types using a single interface called the IPool interface.
```

Comment:
Please expand more on Trident as the is the core thing that we want to cover https://github.com/sushiswap/trident